benchmark/src/main/java/com/turo/pushy/apns/ApnsClientBenchmark.java

```diff
@@ -52,6 +52,9 @@
     @Param({"10000"})
     public int notificationCount;
 
+    @Param({"1", "4", "8"})
+    public int concurrentConnections;
```

Comment:
For the curious, early benchmarks show things to be maaaaaaaaybe slightly faster (it's pretty much in the noise) than they were before channel pooling with a single connection. Unsurprisingly, a single client's throughput goes up significantly when using multiple threads.